mappings/net/minecraft/server/WorldGenerationProgressLogger.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS vg net/minecraft/server/WorldGenerationProgressLogger
+	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD d startTime J
+	FIELD e nextMessageTime J
+	METHOD a start (Lbbt;)V
+		ARG 1 spawnPos
+	METHOD a setChunkStatus (Lbbt;Lbrx;)V
+	METHOD b stop ()V
+	METHOD c getPercentageDone ()I
```

Comment:
`getPercentageDone` -> `getProgress` imo. No need to specify the percentage in the context, and "progress" typically refers to a percentage or completion out of a max amount anyway. The method body is self explanatory.

Reply:
Progress is usually 0-1. Percentage makes it clear it's 0-100.

Reply:
getProgressPercentage